manifests/charts/istiod-remote/templates/mutatingwebhook.yaml

```diff
@@ -16,8 +16,8 @@ webhooks:
   - name: sidecar-injector.istio.io
     clientConfig:
       {{- if .Values.istiodRemote.injectionURL }}
-      # the url doesn't take query params, so use inject/cluster/x/net/y format
-      url: {{ .Values.istiodRemote.injectionURL }}/cluster/{{ .Values.global.multiCluster.clusterName | default "remote0" }}/net/{{ .Values.global.network | default "network0" }}
+      # the url supports to use inject/cluster/x/net/y format
```

Comment:
Would be good to say why someone uses the inject/cluster/x/net/y format?  Seems  it is not something users need to worry about??